src/makeCalculator.js

```diff
@@ -37,7 +37,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    operate(callback, number) {
+      this.result = callback(this.result, number);
+
+      return this;
+    },
+
+    add(result, number) {
+      return result + number;
+    },
```

Comment:
no needs to pass result, just use this. Apply for other calculation methods as well
```suggestion
    add(number) {
      this.result += number;
    },
```